src/db/migrate.rs

```diff
@@ -325,6 +325,17 @@ pub fn migrate(version: Option<Version>, conn: &Connection) -> CratesfyiResult<(
                 ALTER TABLE releases ALTER COLUMN doc_targets DROP NOT NULL;
             "
         ),
+        migration!(
+            context,
+            // version
+            13,
+            // description
+            "Add a field for compression",
+            // upgrade query
+            "ALTER TABLE files ADD COLUMN compressed BOOLEAN NOT NULL DEFAULT false;",
+            // downgrade query
+            "ALTER TABLE files DROP COLUMN compressed;",
+        ),
```

Comment:
I still think that storing this in the s3 header is the way to go

Reply:
For S3 yes, this is just for the database storage backend.